gapis/api/vulkan/transform_command_disabler.go

```diff
@@ -314,10 +370,10 @@ func (disablerTransform *commandDisabler) getNewCommandBufferAndBegin(ctx contex
 	}
 
 	beginInfo := NewVkCommandBufferBeginInfo(inputState.Arena,
-		VkStructureType_VK_STRUCTURE_TYPE_COMMAND_BUFFER_BEGIN_INFO, // sType
-		NewVoidᶜᵖ(memory.Nullptr),                                   // pNext
-		0,                                                           // flags
-		NewVkCommandBufferInheritanceInfoᶜᵖ(memory.Nullptr), // pInheritanceInfo
+		VkStructureType_VK_STRUCTURE_TYPE_COMMAND_BUFFER_BEGIN_INFO,                                         // sType
+		NewVoidᶜᵖ(memory.Nullptr),                                                                           // pNext
+		VkCommandBufferUsageFlags(VkCommandBufferUsageFlagBits_VK_COMMAND_BUFFER_USAGE_ONE_TIME_SUBMIT_BIT), // flags
+		NewVkCommandBufferInheritanceInfoᶜᵖ(memory.Nullptr),                                                 // pInheritanceInfo
```

Comment:
For secondary command buffers, this pInheritanceInfo cannot be NULL. Shouldn't this be the inheritance info of `existingCommandBuffer` ?

Reply:
Yes this was the case I was mentioning in private chat. I am looking into this. This logic might be incorrect in multiple places outside of this transform.

Reply:
I have fixed this. Actually we have to check the other places that this is missing e.g. Vulkan terminator